spring-example/src/test/java/com/apollographql/federation/springexample/AppTest.java

```diff
@@ -0,0 +1,31 @@
+package com.apollographql.federation.springexample;
+
+import com.graphql.spring.boot.test.GraphQLResponse;
+import com.graphql.spring.boot.test.GraphQLTest;
+import com.graphql.spring.boot.test.GraphQLTestTemplate;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@RunWith(SpringRunner.class)
+@GraphQLTest
+public class AppTest {
+    @Autowired
+    private GraphQLTestTemplate graphqlTestTemplate;
+
+    @Test
+    public void lookupPlanckProduct() throws IOException {
+        final GraphQLResponse response = graphqlTestTemplate.postForResource("LookupPlanckProduct.graphql");
+        assertNotNull(response, "response not null");
```

Comment:
nit: This might end up in a double negative if it fails. Might be better to use the positive `response is defined` here.

Reply:
What do you think of the new messages?